app/main.py

```diff
@@ -2,0 +1,55 @@
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
```

Comment:
Appending to `skills` here will modify the class variable shared across all instances. If `skills` were an instance variable, this would not be an issue.